themes/src/main/resources-community/theme/base/account/messages/messages_nl.properties

```diff
@@ -12,6 +22,20 @@ sessionsHtmlTitle=Sessies
 accountManagementTitle=Keycloak Accountbeheer
 authenticatorTitle=Authenticator
 applicationsHtmlTitle=Toepassingen
+
+accountManagementWelcomeMessage=Welkom bij Keycloak Account Management
+personalInfoIntroMessage=Beheer uw basisgegevens
+accountSecurityTitle=Account Beveiliging
+accountSecurityIntroMessage=Beheer uw wachtwoord en account toegang
+applicationsIntroMessage=Volg en beheer uw app-toestemming voor toegang tot uw account
+resourceIntroMessage=Deel uw bronnen met uw team
```

Comment:
are you sure about translating `resources` to "bronnen" maybe better would be "middelen"

Reply:
Yea I've thought about that, not sure what the optimal translation would be. Same thing for Clients, which I did leave at Client(s) as Cliënten feels wrong